blocks/initialization.py

```diff
@@ -159,6 +159,9 @@ class Orthogonal(NdarrayInitialization):
     Only works for 2D arrays.
 
     """
+    def __init__(self, scale=1):
```

Comment:
Please add the new parameters to the docstring. Literature reference would be appreciated as well.